frontend/src/components/explorer/gemBrowser/Metabolite.vue

```diff
@@ -67,7 +67,7 @@
               </div>
               <ExtIdTable :type="type" :external-dbs="metabolite.externalDbs"></ExtIdTable>
             </div>
-            <div v-if="metabolite && metabolite.externalDbs && metabolite.externalDbs.ChEBI"
+            <div v-if="chebiImagePresent"
```

Comment:
What do you think about using something called `chebiImageLink` instead? It could be set as null by default and then in the `setup` method you could set it equal to the link if we get data back. Something like:

```javascript
const link = `https://www.ebi.ac.uk/chebi/displayImage.do?defaultImage=true&imageIndex=0&chebiId=${this.metabolite.externalDbs.ChEBI[0].id.slice(6)}`;
const { data } = await axios.get(link);
if (data !== '') {
  this.chebiImageLink = link;
}
```

This can then of coursed be reused in the img's `src` as well. I was originally going to suggest including `dimensions=400` as well in `setup` but it looks like adding that param triggers a CORS error.

Reply:
Really nice idea! Made a commit incorporating this, was able to add dimension parameter as well

Reply:
Nicely done, much better to have the link concatenation in the method!